upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -929,243 +993,203 @@ spec:
                   containerName: controller
                   divisor: "0"
                   resource: limits.memory
-      affinity:
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-            - matchExpressions:
-              - key: kubernetes.io/os
-                operator: In
-                values:
-                - linux
-              - key: karpenter.sh/provisioner-name
-                operator: DoesNotExist
-      tolerations:
```

Comment:
Why is this being removed?

Reply:
https://github.com/kubernetes/kops/blob/5656e2f37983a99565d264e5542f00ea552665f9/upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template#L1021-L1027

Reply:
Fixed
 - https://github.com/kubernetes/kops/pull/15144/commits/dee14fb84330644298db6daa3c18840954b963f7